spec/services/decisions/income_decision_tree_spec.rb

```diff
@@ -85,6 +85,27 @@
         allow(form_object).to receive_messages(income_above_threshold: YesNoAnswer::NO)
       end
 
+      it { is_expected.to have_destination(:income_savings_assets, :edit, id: crime_application) }
+    end
+  end
+
+  context 'when the step is `income_savings_assets`' do
+    let(:form_object) { double('FormObject') }
+    let(:step_name) { :income_savings_assets }
+
+    context 'when income is above the threshold' do
```

Comment:
Does there need to be a 'when income is below threshold' test as well? (i.e. negative path)

Reply:
A acually that descriptor was incorrect it should have read 'when they have frozen income or assets' copy paste error 😭